common/app/views/fragments/commercial/contentLogo.scala.html

```diff
@@ -1,33 +1,35 @@
-@(branding: common.commercial.Branding, isInteractivePage: Boolean = false)(implicit request: RequestHeader)
+@(branding: common.commercial.Branding, isInteractivePage: Boolean, isCardBranding: Boolean, isOnTheLeft: Boolean)(implicit request: RequestHeader)
 @import common.commercial.PaidContent
 @import implicits.Requests.RichRequestHeader
 
-<div class="brandbadge brandbadge--onside ad-slot--force-display ad-slot--paid-for-badge ad-slot--paid-for-badge--front @if(isInteractivePage) {ad-slot--paid-for-badge--interactive}">
-    <div class="brandbadge__inner ad-slot--paid-for-badge__inner">
-        <h3 class="brandbadge__header">@branding.label</h3>
-        <a href="@branding.sponsorLink" class="brandbadge__link ad-slot--paid-for-badge__link">
-            @if(request.isAmp) {
-                @branding.sponsorLogo.dimensions.map { dim =>
-                    <amp-img src="@branding.sponsorLogo.url"
-                        width="@dim.width"
-                        height="@dim.height"
-                        alt="@{branding.sponsorName}'s logo"
-                        class="brandbadge__logo"></amp-img>
-                }.getOrElse {
-                    <p>@branding.sponsorName</p>
-                }
-            } else {
-                <img src="@branding.sponsorLogo.url"
-                    @for(dim <- branding.sponsorLogo.dimensions) {
-                        width="@dim.width"
-                        height="@dim.height"
-                    }
-                    alt="@{branding.sponsorName}'s logo"
-                    class="brandbadge__logo">
+<div class="adverts__badge js-badge
```

Comment:
Also, because badges can appear in other contexts, what do you think of dropping the `adverts__`? It seems to me that it's a bona fide component.